docs/FAQ.md

```diff
@@ -0,0 +1,106 @@
+
+If you require a specific operator that is not currently supported, please consider [contributing](./../CONTRIBUTING.md) and/or [file an issue](https://github.com/microsoft/onnxruntime/issues) clearly describing your use case and share your model if possible. 
+
+### TensorRT vs CUDA?
+TensorRT and CUDA are separate execution providers for ONNX Runtime. On the same hardware, TensorRT will generally provide better performance; however, this depends on the specific model and whether the operators in the model can be supported by TensorRT. In cases where TensorRT cannot handle the subgraph(s), it will fall back to CUDA. Note that the TensorRT EP may depend on a different version of CUDA than the CUDA EP. 
+
+### TensorRT/CUDA vs DirectML? 
+DirectML is the hardware-accelerated DirectX 12 library for machine learning on Windows and supports all DirectX 12 capable devices (Nvidia, Intel, AMD). This means that if you are targeting Windows GPUs, using the DirectML Execution Provider is likely your best bet. This can be used with both the ONNX Runtime as well as [WinML APIs](./WinRT_API.md).
+
+## Do the GPU builds support quantized models?
+The default CUDA build does not support any quantized operators right now. The TensorRT EP has limited support for INT8 quantized ops. In general, support of quantized models through ORT is continuing to expand on a model-driven basis. For performance improvements, quantization is not always required, and we suggest trying alternative strategies to [performance tune](./ONNX_Runtime_Perf_Tuning.md) before determining that quantization is necessary.
+
+## Why is the GPU build slower than CPU?
+Depending on which execution provider you're using, it may not have full support for all the operators in your model. Fallback to CPU ops can cause hits in performance speed. Moreover even if an op is implemented by our CUDA execution provider, we may not necessarily assign/place the op to the CUDA EP due to performance reasons. To see the placement decided by ORT, turn on verbose logging and look at the console output.
+
+See:
+* [Issue 1368](https://github.com/microsoft/onnxruntime/issues/1368)
+* [Issue 1246](https://github.com/microsoft/onnxruntime/issues/1246)
+
+## My converted Tensorflow model is so slow - why?
+
+NCHW and NHWC are two different memory layout for 4-D tensors.
+
+Most TensorFlow operations used by a CNN support both NHWC and NCHW data format. The Tensorflow team suggests that on GPU NCHW is faster but on CPU NHWC is sometimes faster in Tensorflow. However, ONNX only supports NCHW. As a result, if the original model is in NHWC format, when the model is converted extra transposes may be added. The [tensorflow-onnx](https://github.com/onnx/tensorflow-onnx) and [keras-onnx](https://github.com/onnx/keras-onnx) converters do remove many of these transposes, but if this doesn't help sufficientl, consider retraining the model using NCHW.
+
+## Does the Python API add any overhead to performance?
```

Comment:
This should be two separate questions. One about how to avoid CPU/GPU copy and one about Python API. Even when using the C and C# APIs, unless you do something extra, you will get the GPU/CPU copy, right? Should show how to prevent that when using those APIs

Reply:
We don't have a way to prevent the "copy as part of Run" with the existing APIs.

Reply:
do either of you have a suggest reworded question?

Reply:
We can split it into 2 questions. 
1. Does the Python API add any overhead to performance? Answer is - No, it doesn't. Do we need to have this question? Has it been asked before? Most of the time I've seen the question below.
2. I'm using the Python APIs. Why is ORT slower than PyTorch when using the GPU? We got the same [question](https://github.com/microsoft/onnxruntime/issues/3452) today.
Answer: When using the GPU provider, inputs and outputs need to be copied from CPU to GPU and vice-versa and the ORT Python API doesn't allow inputs/outputs to be setup for this to be done prior to execution (calling run). Hence this copy needs to be done on the fly as part of execution. This eats into the execution time thereby impacting performance. PyTorch, on the other hand, allows you to setup inputs on the GPU prior to execution.

Reply:
I like the split. The answer to question #2 needs some rewording though

Reply:
I'm going to remove the 1st question and reworded the 2nd one: 
**I'm using the Python APIs on GPU and my model is slower than PyTorch.**
This is likely not an execution latency issue with ONNX Runtime. When using the GPU provider, inputs and outputs need to be copied from CPU to GPU and vice-versa. The current version of the ORT Python API makes this copy during execution, while PyTorch allows these to be set up on the GPU prior to execution. Work is in progress to add support of IOBinding in the Python API that allows copying of inputs to the GPU prior to calling Run.

Reply:
The other general reason why ORT GPU execution might be slower than other runtimes (independent of the copying issue) has to do with the fact that not all ops are implemented by our CUDA provider. I usually recommend folks to set_default_logger_severity(0) to see the node placement in the logs.